### PR TITLE
4862 Enhance docstring for the regular 2D image dims

### LINKED ISSUE #4862: Enhance Documentation of LoadImage to Clarify Image Transposition
**Is your feature request related to a problem? Please describe.**
Image data loaded from non-medical 2D formats like jpeg are transposed by the `ImageReader` class loading them, this is because these images are WH order and typical Pytorch ordering is HW. It's not clear to users that this is happening and can be really confused when using the loaders with natural images or photographs, or even MedNIST data as compared to loading with PIL directly.

**Describe the solution you'd like**
Documentation for `LoadImage` should be updated to state that this occurs for a number of non-medical specific file formats. 

**Describe alternatives you've considered**
Tilting my head to the side any time I visualise an image.

**Additional context**
I had forgotten this was intended behaviour and wasn't helped by documentation.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #4862 .

### Description
This PR enhanced the doc-string of LoadImage according to the suggestion from @ericspod , for the regular 2D image dims.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by ericspod** [comment]
Looks good though we should say that readers in general may reorder dimensions to match CHWD Pytorch ordering. `PILReader` does this but others do something similar like [here](https://github.com/Project-MONAI/MONAI/blob/9858293040039ee6f2ee2cf5b5aa737c288f91cf/monai/data/image_reader.py#L355).

File: monai/transforms/io/dictionary.py [hunk 1]
```diff
@@ -51,6 +51,10 @@ class LoadImaged(MapTransform):
         - Current default readers: (nii, nii.gz -> NibabelReader), (png, jpg, bmp -> PILReader),
           (npz, npy -> NumpyReader), (dcm, DICOM series and others -> ITKReader).
 
+    Please note that for png, jpg, bmp format images, `PILReader` will swap axis 0 and 1 after
```
Comment (ericspod) [inline monai/transforms/io/dictionary.py:54]: "...bmp, and other 2D formats, readers often swap..."

File: monai/transforms/io/dictionary.py [hunk 2]
```diff
@@ -51,6 +51,10 @@ class LoadImaged(MapTransform):
...
+    loading the array because the `HW` definition for non-medical specific file formats in PIL is
```
Comment (ericspod) [inline monai/transforms/io/dictionary.py:55]: ~in PIL~

**Review by ericspod — APPROVED** [review]
Minor changes but it's a good reminder to people, thanks!

File: monai/transforms/io/dictionary.py (hunk 1 above)
Comment (Nic-Ma) [reply monai/transforms/io/dictionary.py]: Updated in the latest commit.

Thanks.

File: monai/transforms/io/dictionary.py (hunk 2 above)
Comment (Nic-Ma) [reply monai/transforms/io/dictionary.py]: updated in the latest commit.

Thanks.

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

Thanks for your review.
I updated according to your comments, I didn't put "readers in general may reorder dimensions to match CHWD Pytorch ordering. PILReader does this but others do something similar like here.", because I feel "match CHWD PyTorch ordering" is  slightly confusing..

Thanks.

**Pull request merged**